tests/unit/conftest.py

```diff
@@ -38,7 +38,7 @@ def portainer_stacks(
         },
         {
             "Id": randint(1, 10),
-            "Name": faker.name(),
+            "Name": faker.name().replace(" ", "").lower(),
```

Comment:
Maybe as you did below, `faker.word()` works better here?

Reply:
will put it in thx